content/sql/concepts/primary-keys/primary-keys.md

```diff
@@ -12,7 +12,7 @@ CatalogContent:
   - 'paths/analyze-data-with-sql'
 ---
 
-SQL tables sometimes have a column that uniquely identifies each row of that table. These special columns are called primary keys.
+Sometimes, SQL tables are distinguished through **primary keys**, a special column that will uniquely identify each row of that table.
```

Comment:
```suggestion
**Primary keys** are special columns are used to uniquely identify each row of a table in SQL.
```

Reply:
Hey Dusch! I Guess you meant: "are special columns **that** are used". I will modify and submit accordingly! Thanks!